snippets/telco-core_ClusterLogSubscription.yaml

```diff
@@ -4,7 +4,7 @@ metadata:
   name: cluster-logging
   namespace: openshift-logging
 spec:
-  channel: "stable"
+  channel: "stable-6.0"
   name: cluster-logging
   source: redhat-operators-disconnected
   sourceNamespace: openshift-marketplace
```

Comment:
worth also update `installPlanApproval` to `Manual`

Reply:
It should be there(https://github.com/openshift-kni/telco-reference/pull/32), but let's wait for the containers to pick up the change, thanks